crates/ruff_annotate_snippets/src/renderer/display_list.rs

```diff
@@ -349,10 +363,15 @@ impl DisplaySet<'_> {
                     buffer.puts(line_offset, code_offset, &code, Style::new());
                     if self.margin.was_cut_left() {
                         // We have stripped some code/whitespace from the beginning, make it clear.
-                        buffer.puts(line_offset, code_offset, "...", *lineno_color);
+                        buffer.puts(line_offset, code_offset, cut_indicator, *lineno_color);
                     }
                     if was_cut_right {
-                        buffer.puts(line_offset, code_offset + taken - 3, "...", *lineno_color);
+                        buffer.puts(
+                            line_offset,
+                            code_offset + taken - cut_indicator.width(),
```

Comment:
This looked fishy because the `offset` naming suggests that we operate on byte offsets here but I think this is correct (I also agree that we shouldn't do any renames just to keep the diff with upstream small).

Reply:
Yes. Agreed. Extremely fishy.